pkgs/applications/misc/gostatic/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, nix-update-script
+, testers
+, gostatic
+}:
+
+buildGoModule rec {
+  pname = "gostatic";
+  version = "2.34";
+
+  src = fetchFromGitHub {
+    owner = "piranha";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-rdbiIFRZcn9dVaF2anl2iy6FM6boz38vjn+hCpMwcis=";
+  };
+
+  vendorSha256 = "sha256-9YCt9crLuYjd+TUXJyx/EAYIMWM5TD+ZyzLeu0RMxVc=";
+
+  ldflags = [ "-s" "-w" ];
+
+  passthru = {
+    updateScript = nix-update-script { attrPath = pname; };
```

Comment:
If I'm not mistaken, this part is not actually required for @r-ryantm to update the package as `nix-update` can work without it. I could be wrong, though.

Reply:
@ryantm Do you mind confirming if this is true or not?

Reply:
That's true, but I think it is still good practice to have an passthru.updateScript because it documents how to update it, gives more control to the maintainers, and is potentially useful for other tools too.